docs/dev_guide/diagnostic_ref/dpct1056.rst

```diff
@@ -26,3 +26,47 @@ If this variable is only used in host code, ignore this warning.
 
 If the variable is also used in device code, you need to migrate this code again
 with ``--extra-arg="-xcuda"`` passed to the tool to explicitly specify the code language.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   // dpct --out-root out test.c --extra-arg=-xc
+   
+   // test.c 
+   #include <stdio.h>
+   #include <cuda_runtime.h>
+   
+   static __constant__ const float aaa = 10.f;
+   
+   void foo(){
+     printf("%f\n", aaa);
+   }
+
+results in the following migrated SYCL\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   /*
+   DPCT1056:0: The use of aaa in device code was not detected. If this variable is
+   also used in device code, you need to rewrite the code.
+   */
+   static const float aaa = 10.f;
+   
+   void foo(){
+     printf("%f\n", aaa);
+   }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+   static const float aaa = 10.f;

```

Comment:
Please give a ref code if this variable aaa is used in device code.

Reply:
done